controllers/dspipeline_params.go

```diff
@@ -89,8 +89,8 @@ type DSPAParams struct {
 	CustomCABundle *dspa.CABundle
 	DSPONamespace  string
 	// Use to enable tls communication between component pods.
-	PodToPodTLS bool
-
+	PodToPodTLS             bool
+	CustomExtraParams       string
```

Comment:
It's not 100% clear to me, but I believe TLS should be enabled when `PodToPodTLS` is `true`. Can you confirm @HumairAK?

Reply:
agreed.

so instead of doing this:
```
  database:
    customExtraParams: '{"tls":"true"}'
```

just make that happen automatically when `PodToPodTLS` is `true`

Reply:
also it doesn't look like the value of `customExtraParams` is actually used anywhere, which is a bit puzzling

Reply:
@HumairAK  could you confirm that TLS should be enabled when PodToPodTLS is true. I remember that we discussed to use CustomExtraParams instead of PodToPodTLS in this case, so just want to make sure of that.

Reply:
sorry for the delayed response, yes that's correct, we only want to enable mariadb when podtopodtls is set to true, in this scenario we should probably configure extra params automatically as needed to make this work

Reply:
> in this scenario we should probably configure extra params automatically as needed to make this work

so we don't need the `customExtraParams` at all. Cool. Thanks for confirming.